bithubph/bithub-fe
# #172030011 setup react application

#### What does this PR do?
- This PR sets up the authentication feature of the application

#### Description of tasks to be completed?
- install react using CRA
- add components folder
- add ui, pages folder
- add atoms, molecules and template folders
- setup landing page

#### How should this be manually tested?
  - Clone the repo with `clone https://github.com/bithubph/bithub-fe.git`
  - Checkout to branch `ft-setup-react-application-172030011`
  - Install dependencies `yarn install`
  - Run application `yarn start`
  - An index page is displayed with text: `Welcome to BitHub`

#### Any background context you want to provide?
- You would need npm or yarn to run this application

#### What are the relevant pivotal tracker stories?
- PT Story link - [#172030011](https://www.pivotaltracker.com/story/show/172030011)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/31358867/78120439-dd036d00-7401-11ea-9b1c-3a93ee040084.png)
